sap-s4hana-cloud/2.6/modules/ROOT/pages/sap-s4hana-cloud-connector-reference.adoc

```diff
@@ -139,9 +141,10 @@ include::connectors::partial$oauth2-description.adoc[]
 ** HOURS
 ** DAYS |  Time unit for the *EDM Expiration Interval* field. |  MINUTES |
 | Discovery Service Enabled a| Boolean |  Feature switch for automatic service discovery. |  `true` |
-| Service discovery relative path a| String |  Relative path from the base URL to the endpoint used for automatic services discovery. | `sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ServiceCollection` |
-| Service directory a| String |  Common HTTP directory for all unlisted services. This parameter enables users to specify only service names in the unlisted services list instead of service URLs. It also enables the connector to construct the service URL using the following format: `{base URL parameter}/{this parameter}/{item from unlisted services list}` |   |
+| Scoped service discovery relative path a| String | Relative path from the base URL to the endpoint used for automatic services discovery with custom namespaces on cloud instances. |  `sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ScopedServiceCollection` | 
```

Comment:
The "with custom namespaces on cloud instances" is a bit confusing, but I am not sure if you can clarify it. Is the automatic services discovery done using these namespaces?